web/src/components/NearbyCities.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { getNearbyCities } from 'shared'
+import { CityModel } from 'shared/api'
+
+import { RefreshIcon } from '../assets'
+import useUserLocation from '../hooks/useUserLocation'
+import CityEntry from './CityEntry'
+import { CityListParent } from './CitySelector'
+import Button from './base/Button'
+import Icon from './base/Icon'
+
+const NearbyMessageContainer = styled.div`
+  padding: 8px;
+  flex-direction: row;
+  justify-content: space-between;
+`
+
+const NearbyMessage = styled.span`
+  color: ${props => props.theme.colors.textColor};
+  font-family: ${props => props.theme.fonts.web.decorativeFont};
+  padding-top: 16px;
+`
+
+const RetryButtonContainer = styled(Button)`
+  flex-direction: column;
+  height: 24px;
+`
+
+const StyledMessageWrapper = styled.div`
+  display: flex;
+  align-items: end;
+  justify-content: space-between;
+`
+
+type NearbyCitiesProps = {
+  cities: CityModel[]
+  language: string
+  filterText: string
+  stickyTop: number
+}
+
+const NearbyCities = ({ cities, language, filterText, stickyTop }: NearbyCitiesProps): ReactElement => {
+  const { t } = useTranslation('landing')
+  const { data: userLocation, refresh } = useUserLocation()
+
+  const nearbyCities = userLocation
+    ? getNearbyCities(
+        userLocation,
+        cities.filter(city => city.live),
+      )
+    : []
+
+  return (
+    <>
```

Comment:
:wrench: I am sorry, but seems like we should readd the `div` here. Otherwise we have this weird visual effect:
```suggestion
    <div>
```
![image](https://github.com/user-attachments/assets/b55cfa9f-1c48-4098-be9e-e3bb2ea6f8ff)